cpp/include/cuspatial/vec_2d.hpp

```diff
@@ -53,6 +53,18 @@ std::ostream& operator<<(std::ostream& os, cuspatial::vec_2d<T> const& vec)
   return os << "(" << vec.x << "," << vec.y << ")";
 }
 
+/**
+ * @brief Print function usable in host or device code.
+ */
+template <typename T>
+void CUSPATIAL_HOST_DEVICE print(cuspatial::vec_2d<T> const& vec)
```

Comment:
I don't think we really need this utility other than debugging in the library?

Reply:
Is that not a good reason to have it? We have operator<< already, but that didn't work in device code so I added this.

Reply:
Cool, I think it'll come handy when I need it.

Reply:
Ah, you removed them in the latest commit. I think it's ok either way.